dashboard/client/src/common/CodeDialogButton/CodeDialogButton.tsx

```diff
@@ -111,13 +114,15 @@ export const CodeDialogButtonWithPreview = ({
   const buttonTextToPass = buttonText ?? "Expand";
 
   return (
-    <div className={classNames(classes.root, className)}>
-      <span className={classes.previewText}>{codeText}</span>
+    <Box className={className} sx={Object.assign({}, styles.root, sx)}>
```

Comment:
```suggestion
    <Box className={className} sx={[styles.root, sx]}>
```

Please use [array syntax](https://mui.com/system/getting-started/the-sx-prop/#array-values) instead